mysql/datadog_checks/mysql/config.py

```diff
@@ -119,3 +137,20 @@ def configuration_checks(self):
 
         if self.mysql_sock and self.host:
             self.log.warning("Both socket and host have been specified, socket will be used")
+
+    @staticmethod
+    def _should_propagate_agent_tags(instance, init_config) -> bool:
```

Comment:
Should we consider making this function shared somewhere since I'm copying it over from the postgres integration?